pkg/objectio/object.go

```diff
@@ -16,7 +16,7 @@ package objectio
 
 import "github.com/matrixorigin/matrixone/pkg/fileservice"
 
-const Magic = 0xFFFFFFFF
+const Magic = 0xFFFFFFFFFFFFFFFF
```

Comment:
This is not a good magic value.   It clashes with -1, too often, some code will produce a -1.
Use a strange one ...

Reply:
removed